litefs/disaster-recovery.html.md.erb

```diff
@@ -0,0 +1,174 @@
+---
+title: Disaster Recovery from LiteFS Cloud
+layout: docs
+sitemap: false
+nav: litefs
+---
+
+The biggest advantage of using LiteFS Cloud is that you have backups
+to restore from in case of an unfortunate situation where you lose
+all the nodes in your cluster. In this doc, we'll show you how to restore
+your cluster from LiteFS Cloud if you've lost all the nodes.
+
+<div class="callout">
```

Comment:
I thought about this and originally decided to have it as info rather than warning, because they'll still be able to recover from whichever LiteFS Cloud snapshot they want. But it will be a big hassle, and deleting data is always bad.... so I'll switch to warning!